ddl/disttask_flow.go

```diff
@@ -33,142 +33,82 @@ import (
 )
 
 type litBackfillFlowHandle struct {
-	d DDL
+	d *ddl
 }
 
 var _ dispatcher.TaskFlowHandle = (*litBackfillFlowHandle)(nil)
 
 // NewLitBackfillFlowHandle creates a new litBackfillFlowHandle.
-func NewLitBackfillFlowHandle(d DDL) dispatcher.TaskFlowHandle {
-	return &litBackfillFlowHandle{
-		d: d,
+func NewLitBackfillFlowHandle(d DDL) (dispatcher.TaskFlowHandle, error) {
+	ddl, ok := d.(*ddl)
+	if !ok {
+		return nil, errors.New("The getDDL result should be the type of *ddl")
```

Comment:
Can we log here instead of return the error but ignore?

Reply:
> ignore

I think current implementation is ok, we can't assume the `d DDL` is `*ddl`